### PR TITLE
Add Ember Europe Q1 2023 to calendar

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add file
Screenshot 2023-02-22 at 12 33 57

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
@marcoow since I don't have access to Meetup anymore, I can't see the event description. Hope this is still correct.

**Comment by marcoow** [comment]
you need an account and join the group – it's an open group and everyone can join

File: src/calendar/2023-03-23-embereurope-ghent.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Ember Europe Q1 Meetup"
+image: "/assets/images/calendar/2023-03-23-embereurope-ghent/embereurope.png"
+location: Ghent
+url: https://www.meetup.com/ember-europe/
+kind: meetup
+color: purple
+---
+
+Ember Europe is back in 2023 with the first Meetup of the year the year, hosted
```
Comment (marcoow) [inline src/calendar/2023-03-23-embereurope-ghent.md]:
```suggestion
Ember Europe is back in 2023 with the first Meetup of the year, hosted
```

File: src/calendar/2023-03-23-embereurope-ghent.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+by OTA Insight in Ghent, BE. The Ember Europe meetup series brings together the
```
Comment (marcoow) [inline src/calendar/2023-03-23-embereurope-ghent.md]:
```suggestion
by OTA Insight in Ghent, Belgium. The Ember Europe meetup series brings together the
```

File: src/calendar/2023-03-23-embereurope-ghent.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Ember community from all of Europe once a quarter. The meetup is organized by
+Mainmatter.
```
Comment (marcoow) [inline src/calendar/2023-03-23-embereurope-ghent.md]:
```suggestion
Mainmatter and Intercom.
```

**Review by marcoow — COMMENTED** [review]
…

**Comment by emmasofiah2o** [comment]
@marcoow let me know when to merge

File: src/calendar/2023-03-23-embereurope-ghent.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Ember Europe Q1 Meetup"
+image: "/assets/images/calendar/2023-03-23-embereurope-ghent/embereurope.png"
+location: Ghent
+url: https://www.meetup.com/ember-europe/
+kind: meetup
+color: purple
+---
+
+Ember Europe is back in 2023 with the first Meetup of the year, hosted by OTA
```
Comment (marcoow) [inline src/calendar/2023-03-23-embereurope-ghent.md]:
```suggestion
Ember Europe is back in 2023 with the first meetup of the year, hosted by OTA
```

**Review by marcoow — APPROVED** [review]
one typo but approving already

**Pull request merged**